api-catalog-ui/frontend/src/components/Wizard/configs/wizard_defaults.jsx

```diff
@@ -1,5 +1,11 @@
-/**
```

Comment:
Same as for previous shouldn't we keep the comment after the license?

Reply:
Remove by eslint/header autofix, added comment back